iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/ExportOptions.java

```diff
@@ -51,31 +51,33 @@
     /**
      * Sort IIDM objects so that generated XML does not depend on data model object order. Depending on object types the
      * following sorting key has been chosen:
-     *  - the id for identifiables
-     *  - the name for extensions
-     *  - the name for temporary limits
-     *  - node1 then node2 for internal connections
+     * - the id for identifiables
+     * - the name for extensions
+     * - the name for temporary limits
+     * - node1 then node2 for internal connections
+     * - the name for properties of an identifiable
      */
     private boolean sorted = false;
 
     public ExportOptions() {
     }
 
-    public ExportOptions(boolean withBranchSV, boolean indent, boolean onlyMainCc, TopologyLevel topologyLevel, boolean throwExceptionIfExtensionNotFound) {
-        this(withBranchSV, indent, onlyMainCc, topologyLevel, throwExceptionIfExtensionNotFound, null);
+    public ExportOptions(boolean withBranchSV, boolean indent, boolean onlyMainCc, TopologyLevel topologyLevel, boolean throwExceptionIfExtensionNotFound, boolean sorted) {
+        this(withBranchSV, indent, onlyMainCc, topologyLevel, throwExceptionIfExtensionNotFound, sorted, null);
     }
 
-    public ExportOptions(boolean withBranchSV, boolean indent, boolean onlyMainCc, TopologyLevel topologyLevel, boolean throwExceptionIfExtensionNotFound, String version) {
-        this(withBranchSV, indent, onlyMainCc, topologyLevel, throwExceptionIfExtensionNotFound, version, THROW_EXCEPTION);
+    public ExportOptions(boolean withBranchSV, boolean indent, boolean onlyMainCc, TopologyLevel topologyLevel, boolean throwExceptionIfExtensionNotFound, boolean sorted, String version) {
+        this(withBranchSV, indent, onlyMainCc, topologyLevel, throwExceptionIfExtensionNotFound, sorted, version, THROW_EXCEPTION);
```

Comment:
Please also keep the previous constructors to avoid breaking changes of public constructors